apps/home/templates/sections/quotes.jade

```diff
@@ -0,0 +1,32 @@
+.HomeSection__content.HomeQuotes
+  .HomeSection__cta.HomeQuotes__cta
+    h2.Type.Type--size-h2 What people are saying
+  .HomeQuotes__quotes
+    .HomeQuotes__quote
+      .HomeQuotes__icon
+        img(src="https://d2w9rnfcy7mm78.cloudfront.net/1807149/original_46cd0d14fec88063fb61b5704d3fcbc3.png")
+      .HomeQuotes__copy 
+        | “In the groundswell of anger and suspicion toward social media platforms, #[span Are.na feels like a necessary antidote].”
+        br
+        | - Co.Design
+    .HomeQuotes__quote
+      .HomeQuotes__icon
+        img(src="https://d2w9rnfcy7mm78.cloudfront.net/1807150/original_7d14b1f553ae180878bd802c426c4aaf.png")
+      .HomeQuotes__copy 
+        | “Are.na is building a network of content, ideas, and the people that have them. #[span By removing ads and ego-stroking ‘likes,’ Are.na frees up space for creativity.]”
+        br
+        | - Art.sy
+    .HomeQuotes__quote
+      .HomeQuotes__icon
+        img(src="https://d2w9rnfcy7mm78.cloudfront.net/1807151/original_467984445f140a7eeafa0bec5ab62665.png")
+      .HomeQuotes__copy
+        | “Are.na is one of those rare social-ish networks that puts #[span ideas, calmness, and thoughtfulness ahead of everything else].”
+        br
+        | - Willa Köerner, Creative Content Director at The Creative Independent
+    .HomeQuotes__quote
+      .HomeQuotes__icon
+        img(src="https://d2w9rnfcy7mm78.cloudfront.net/1807152/original_6e0d9aaf81d3ad09bcc788cd7e4db90c.png")
+      .HomeQuotes__copy 
+        | “#[span Are.na’s whole vibe makes you want to be smarter], dig deeper, participate in collective knowledge” 
+        br
+        | - Ben Pieratt, Founder of Svpply
```

Comment:
I'm wondering when I developed a complex about whitespace-significant templating languages.

Reply:
Yeah, jade is awful